### DISCUSSION TITLE
How to restart chia node via cmd?

### DISCUSSION THREAD

**Discussion by elmans1961** [Chia-Network/chia-blockchain#4564] [Support]
1.I have problems with full node  Synced for the last 3 days.I tried restart "all" via cmd but failed.It executes all command chia start,chia show -s  etc.I tied chia -r,,chia restart chia --restart chia node -r but failed.
By the way why chia  Syncing works so unstable?It makes unpossible to "harvest a coin" even having1000 plots not saying of 10 or 50.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
